src/Symfony/Bridge/Monolog/Handler/SwiftMailerHandler.php

```diff
@@ -54,6 +56,10 @@ public function onCliTerminate(ConsoleTerminateEvent $event)
      */
     protected function send($content, array $records): void
     {
+        if (__CLASS__ === static::class) {
+            trigger_deprecation('symfony/monolog-bridge', '5.4', '"%s" is deprecated and will be removed in 6.0.', __CLASS__);
+        }
```

Comment:
The `send()` method is called when a message is logged and `trigger_deprecation()` might trigger a log message. This might result in a nasty recursion.

Shall we move this block to the constructor instead?

Reply:
or above the constructor?